service.upnext/resources/lib/player.py

```diff
@@ -31,7 +32,7 @@ def disable_tracking(self):
 
     def onPlayBackStarted(self):  # pylint: disable=invalid-name
         """Will be called when kodi starts playing a file"""
-        Monitor().waitForAbort(5000)
+        self.monitor.waitForAbort(5)
```

Comment:
ups, missed this one. Thanks for the quick update! waitForAbort uses seconds.

Reply:
yeah me too! cheers